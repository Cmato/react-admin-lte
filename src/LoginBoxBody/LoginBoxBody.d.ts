import * as React from 'react';

export interface LoginBoxBodyProps {
}

declare const LoginBoxBody: React.ComponentType<LoginBoxBodyProps>;

export default LoginBoxBody;
