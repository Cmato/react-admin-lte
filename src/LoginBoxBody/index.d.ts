export { default } from './LoginBoxBody';
export * from './LoginBoxBody';
