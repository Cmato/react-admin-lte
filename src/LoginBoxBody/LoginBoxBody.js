import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';

const LoginBoxBody = ({
  children,
  className: classNameProp,
  ...other
}) => {
  const className = classNames({
    'login-box-body': true
  }, classNameProp);

  return (
    <div
      className={className}
      {...other}
    >
      {children}
    </div>
  );
};

LoginBoxBody.propTypes = {
  /**
   * The contents of the component.
   */
  children: PropTypes.node.isRequired,
  /**
   * @ignore
   */
  className: PropTypes.string
};

export default LoginBoxBody;
