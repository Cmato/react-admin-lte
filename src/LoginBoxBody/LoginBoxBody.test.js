import React from 'react';
import { shallow } from 'enzyme';
import LoginBoxBody from './LoginBoxBody';

it('renders correctly', () => {
  expect(shallow(<LoginBoxBody><div /></LoginBoxBody>))
    .toMatchSnapshot();
});
