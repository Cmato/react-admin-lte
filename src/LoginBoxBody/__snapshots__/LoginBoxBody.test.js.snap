// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="login-box-body"
>
  <div />
</div>
`;
